### PR TITLE
fix: updated the UI of the progress bar in podcast

### LINKED ISSUE #451: Improve the UI of the progress bar in podcast episode view
Currently the progress bar works but during the initial few seconds the ball doesn't move from the left end. This may confuse users as to whether the progress bar is working or not.

Here's a [link](https://github.com/freeCodeCamp/mobile/compare/653ad88e54d45c574615d0906584b947179ec012..c6661040f268607b67ea5898a941d72675877a2a) with an attempt to fix it but since no progress was being made we reverted back.

### PR DESCRIPTION
**Pull request by Dexters-Hub**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #451

Screenshot of the changed code is attached below:
Screenshot from 2022-08-04 23-58-47

**Problems:** 
- Basically in the previous code the ball only moves after 42 seconds.
-  The ending of the podcast and the progress bar was not in sync (either the bar stops early, or there is still left of the podcast)

**Solution:** 
- By dividing with larger number the seconds will be reduced. I have made it to 14 seconds.
- Calculated the amount of newWidth and multiplied with 0.95

### CODE REVIEW

**Review by Sembauke — APPROVED** [review]

**Comment by Sembauke** [comment]
Hi @Dexters-Hub  Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝
